NewForce-Cohort-5/tabloidmvc-the-spaghetti-squad
# Ticket #29 Register

# Description

New users should be able to register for an account and create a "UserProfile" using a Form.  The new user will be able to input User Profile information into a form (First Name, Last Name, Email, and Display Name), establish an account with all included information (First Name, Last Name, Email, and Display Name, CreationDateTime, Full Name, UserTypeId), and be logged in. The user is assigned a User Type of "Author" automatically and creation time is set to capture when the form is submitted.

Link to ticket: https://trello.com/c/QFXFFy4l

## Type of change

Please delete options that are not relevant.


# Testing Instructions

testing objectives

to create a new User Profile 
to automatically be assigned a UserTypeId and CreationDateTime
to be immediately logged in an directed to home page

testing instructions

If you currently do not have access to this branch, go ahead and run a <git fetch --all>, then to navigate to this branch jt-register to test

Navigate to register link to populate form

Input required information (Firstname, LastName, Email, DisplayName)

Submit form and watch for the redirection to home page making sure you are logged in to that new user

Run a new query `SELECT * FROM UserProfile` and check the table for the new user - specifically creation time and UserTypeId






# Checklist:
